lib/std/io/writer.zig

```diff
@@ -34,6 +34,12 @@ pub fn Writer(
             return std.fmt.format(self, format, args);
         }
 
+        pub fn printAndCount(self: Self, comptime format: []const u8, args: anytype) Error!u64 {
```

Comment:
Should be `usize` to match `write`:

```suggestion
        pub fn printAndCount(self: Self, comptime format: []const u8, args: anytype) Error!usize {
```

If you want to make an argument that they should all be changed to `u64` we can have that discussion but should be a different issue.

Reply:
Now this begs the question... what if we print something that is larger than the platforms `usize`?

Reply:
How would you do that?

Reply:
in 2 (or really: many) write calls.

e.g. imagine you have a `[1024*1024*1024*2]u8` (2GB array) and you print it with `{}` ==> you'd print off 3-5 characters per entry: for a total of 6-10GB of output.

Reply:
In Zig we have the precedent that `usize` + `usize` is still another `usize`:

```zig
test {
    var a: usize = foo();
    var b: usize = bar();
    var c = a + b; // the type will be usize
    var d = a * b; // even for this one
}
```

same idea here with `write()` multiple times.

Reply:
Of course. I think the issue here is that printing can lead to a number that may be more than what a platform decides a `usize` is